### PR TITLE
Source this exact version of solidus_social in readme

### PR DESCRIPTION
**Pull request by jgujgu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Use with caution.
 
 Add this extension to your `Gemfile`:
 ```ruby
-gem 'solidus_social'
+gem 'solidus_social', git: 'git://github.com/solidusio-contrib/solidus_social.git'
```
Comment (tvdeyen) [inline README.md]: You can write this shorter with using the `github` shortcut of bundler

``` ruby
gem 'solidus_social', github: 'solidusio-contrib/solidus_social'
```

**Comment by tvdeyen** [comment]
Thanks.

**Comment by cbrunsdon** [comment]
Thanks @jgujgu, but I've now released 1.0 to rubygems so I'd prefer people use that than the github URI, especially since its under the "setup for production" header.

**Comment by jgujgu** [comment]
Very reasonable. This gem is moving up in life.

**Pull request closed without merging**